doc/en/components/grids/_shared/export-excel.md

```diff
@@ -0,0 +1,234 @@
+
+> [!NOTE]
+> The exported {Component} will not be formatted as a table, since Excel tables do not support multiple row headers.
+
+<!-- ComponentStart: Grid -->
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/grid/multi-column-headers-export" alt="{Platform} Multi-Column Headers Export">
+</code-view>
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/treegrid-multi-column-headers-export" alt="{Platform} Multi-Column Headers Export">
+</code-view>
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-multi-column-export" alt="{Platform} Multi-Column Headers Export">
+</code-view>
+<!-- ComponentEnd: HierarchicalGrid -->
+
+## Export Grid with Frozen Column Headers
+
+By default Excel Exporter service exports the grid with scrollable (unfrozen) column headers. There are scenarios in which you may want to freeze all headers on top of the exported excel file so they always stay in view as the user scrolls through the records. To achieve this you could set the [exporter option]({environment:demosBaseUrl}/classes/exporteroptionsbase.html) [freezeHeaders]({environment:demosBaseUrl}/classes/exporteroptionsbase.html#freezeHeaders) to `true`.
+
+```razor
+public exportButtonHandler() {
+    const exporterOptions = new ExcelExporterOptions('ExportedDataFile');
+    exporterOptions.freezeHeaders = true;
+    this.excelExportService.export(this.grid, exporterOptions);
+}
+```
+
+## Customizing the Exported Content
+
+In the above examples the Excel Exporter service was exporting all available data. There are situations in which you may want to skip exporting a row or even an entire column. To achieve this you may hook to the [`columnExporting`]({environment:demosBaseUrl}/classes/excelexporterservice.html#columnexporting) and/or [`rowExporting`]({environment:demosBaseUrl}/classes/excelexporterservice.html#rowexporting) events which are fired respectively for each column and/or each row and cancel the respective event by setting the event argument object's [`cancel`]({environment:demosBaseUrl}/interfaces/irowexportingeventargs.html#cancel) property to `true`.
```

Comment:
remove all hard-code links and just use API members (in camelcase) and wrap them in backticks, e.g. ``` `ExcelExporterService` ``` ``` `Export` ```